app/controllers/api/v1/meetupsController.js

```diff
@@ -1,10 +1,10 @@
 // imports
-const { validationResult } = require('express-validator/check');
-const meetups = require('./../../../dataStore/meetupsTable');
-const rsvps = require('./../../../dataStore/rsvpsTable(meetups-tables)');
+import { validationResult } from 'express-validator/check';
+import meetups from '../../../dataStore/meetupsTable';
+import rsvps from '../../../dataStore/rsvpsTable(meetups-tables)';
 
 
-module.exports = {
+export default {
     index(req, res) {
```

Comment:
You should refactor this to es6 also

Reply:
@SannyTee Thanks so much. They have actually been refactored. The line above, which is the module.exports has been converted to the line beneath, which is  export default.